app/src/main/res/values-de/strings.xml

```diff
@@ -48,6 +48,8 @@
   <string name="basic_sensor_name_location_accurate">Einzelne genaue Position</string>
   <string name="basic_sensor_name_location_background">Hintergrund Standort</string>
   <string name="basic_sensor_name_location_zone">Standortzone</string>
+  <string name="basic_sensor_name_mobile_data">Mobile Daten</string>
```

Comment:
Please remove these strings from this time. We use lokalize which updates these strings. Only English needs strings here.

Reply:
Done.